core/base/inc/CommandLineOptionsHelp.h

```diff
@@ -66,6 +66,7 @@ Usage: %s [-l] [-b] [-n] [-q] [dir] [[file:]data.root] [file1.C ... fileN.C]
   -q : exit after processing command line macro files
   -l : do not show splash screen
   -t : enable thread-safety and implicit multi-threading (IMT)
+ -web: display graphics in a web browser
```

Comment:
Maybe `--web` to follow the `--notebook` precedent?

Reply:
Sure.